__tests__/generate.spec.ts

```diff
@@ -64,8 +64,8 @@ describe('Generate', () => {
 
 
       #################################### Generated content - do not edit! ####################################
-      # This block has been generated with codeowners-generator (for more information https://github.com/gagoar/codeowners-generator/README.md)
-      # To re-generate, run \`yarn codeowners-generator generate\`. Don't worry, the content outside this block will be kept.
+      # This block has been generated with codeowners-generator (for more information https://github.com/gagoar/codeowners-generator)
```

Comment:
can you add make one of these tests change to include the same legend it had before? I would like to have the use case.
We actually could add in the readme something to this intent. 

I'm trying to keep the status quo as similar as we can.